src/partials/60_section-gallery.html

```diff
@@ -273,43 +465,72 @@ <h5 class="product-title">Aenean Ru Bristique</h5>
                   <p class="old-price"><s>$160.00</s></p>
                 </div>
               </div>
+              <div class="gallery-slider-wrapper">
+                <div class="gallery-slider">
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/sofa-1.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/sofa-2.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/sofa-3.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/sofa-4.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/bed.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/chair-12.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/chair-1.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/sofa-2.png);"
+                    ></div>
+                  </div>
+                  <div>
+                    <div
+                      class="slider-image"
+                      style="background-image: url(images/gallery/chair-5.png);"
+                    ></div>
+                  </div>
+                </div>
+              </div>
+              <!--  /slider -->
             </div>
+            <!--  /tab-pane -->
+            <div class="fade-layer"></div>
           </div>
-          <div class="slider">
```

Comment:
czego dotyczy ten kod i dlaczego go usuwasz? przez to pojawia sie blad w konsoli, poniewaz js nie moze znalezc elementu `.slider`

Reply:
Ten slider (a raczej coś co tylko wizualnie wyglądało jak slider) wywalam, bo mi jest już nie potrzebny. Zamiast niego JS generuje 4 osobne slidery  dla każdego taba (w divach z klasą `.gallery-slider`).

Reply:
to musisz zaktualizowac jakos klasy w html/js, bo aktualnie nie wszystkie `.tab-pane` na stronie zawieraja w sobie `.gallery-slider`

Reply:
?? Z tego co widzę, to wszystkie. W galerii są cztery `.tab-pane` i w każdym z nich jest `.gallery-slider`...

Reply:
Chyba, że chodzi o taby z sekcji z nowymi meblami. Tam dominik też zastosował taby, no ale sliderów tam nie powinno być...